src/webvr.config.js

```diff
@@ -1,51 +0,0 @@
-export default {
```

Comment:
Did WebVR Boilerplayer not look for it globally? Or is it because the config was being set too late? (See my comment about import ordering)